ui/src/alerting/constants/index.ts

```diff
@@ -79,11 +79,11 @@ export const DEFAULT_ENDPOINT_URLS = {
 export const NEW_ENDPOINT_DRAFT: NotificationEndpoint = {
   name: 'Name this Endpoint',
   description: '',
-  status: 'active',
+  activeStatus: 'active',
   type: 'slack',
   token: '',
   url: DEFAULT_ENDPOINT_URLS['slack'],
-  loadingStatus: RemoteDataState.Done,
+  status: RemoteDataState.Done,
```

Comment:
oof- this is going to be super confusing someday.  but maybe types will make it ok?

Reply:
yea, it was definitely giving me a headache.  but I think the `activeStatus` and `status` with proper typing make it much more clear.